### ISSUE TITLE
finish-args-redundant-device-all not useful

### ISSUE THREAD

**Issue by hadess** [flathub/flatpak-builder-lint#54]
https://github.com/flathub/org.gnome.Totem.Devel/blob/beta/codecs/ffmpeg.json uses:
```
        /* OpenGL access */
        "--device=dri",
        /* DVD playback */
        "--device=all",
```
Which triggers the error `finish-args-redundant-device-all`. But the `--device` while redundant is a good documentation. The error should warn for duplicate `--device` flags instead.

**Comment by hadess** [comment]
How long should it take for that change to deploy to Flathub? That's still throwing errors on Flathub this afternoon.
